packages/maskbook/src/resources/AirdropIcon.tsx

```diff
@@ -0,0 +1,133 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core'
+
+const AirdropIconSVG = (
+    <svg viewBox="0 0 70 79" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M35.523 70.684l-12.818-1.13 4.489-11.115 12.817 1.13-4.488 11.115z" fill="url(#paint0_linear)" />
```

Comment:
this `id` may conflict
 
https://github.com/DimensionDev/Maskbook/blob/36b422fab16b09f9a06a8c8d3104651a12227e59/packages/maskbook/src/plugins/Wallet/UI/WalletConnectQRCodeDialog/Icons.tsx#L26
https://github.com/DimensionDev/Maskbook/blob/36b422fab16b09f9a06a8c8d3104651a12227e59/packages/maskbook/src/plugins/Wallet/UI/WalletConnectQRCodeDialog/Icons.tsx#L38-L50